packages/next-app/package.json

```diff
@@ -22,7 +22,9 @@
     "@hookform/resolvers": "^3.0.0",
     "@pinata/sdk": "^2.1.0",
     "@radix-ui/colors": "^0.1.8",
+    "@radix-ui/react-accordion": "^1.1.1",
```

Comment:
When I first saw this I thought of using their `Collapsible` instead because I thought  Accordion only allowed for one of the items to be open at once. But it sounds like there's a `type="multiple"` prop you can pass to it! 😄